CITATION.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Stone"
+  given-names: "Connor"
+  orcid: "https://orcid.org/0000-0002-9086-6398"
+- family-names: "Adam"
+  given-names: "Alexandre"
+  orcid: "https://orcid.org/0000-0001-8806-7936"
+- family-names: "Perreault-Levasseur"
+  given-names: "Laurence"
+  orcid: "https://orcid.org/0000-0003-3544-3939"
+- family-names: "Hezaveh"
+  given-names: "Yashar"
+  orcid: "https://orcid.org/0000-0002-8669-5733"
+- family-names: "Core"
+  given-names: "Cordero"
+  orcid: "https://orcid.org/0000-0002-3531-3221"
+- family-names: "Setiawan"
+  given-names: "Don"
+  orcid: "https://orcid.org/0000-0002-1624-2667"
+- family-names: "Coogan"
+  given-names: "Adam"
+- family-names: "Filipp"
+  given-names: "Andreas"
+- family-names: "Wilson"
+  given-names: "Charles"
+- family-names: "Yantovski-Barth"
+  given-names: "Misha"
+- family-names: "Legin"
+  given-names: "Ronan"
+title: "caustics"
+doi: 10.5281/zenodo.10806383
+version: v0.9.0
+date-released: 2024-03-11
```

Comment:
```suggestion
# Update below with the appropriate
# zenodo DOI, version, and release date
# for each release
doi: 10.5281/zenodo.10806383
version: v0.9.0
date-released: 2024-03-11
```

Reply:
Will we need to update with every release? Is there a way to automate this?

Reply:
```suggestion
doi: 10.5281/zenodo.10806382
abstract: "The lensing pipeline of the future: GPU-accelerated, automatically-differentiable, highly modular. Currently under heavy development: expect interface changes and some imprecise/untested calculations."
repository-code: "https://github.com/Ciela-Institute/caustics"
```

If you want to automate it, then let's not specify a version, but rather cite for all versions

Reply:
@uwcdc please modify to the second suggestion above

Reply:
Also, remove the comments